src/style.css

```diff
@@ -1 +1,58 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc');
+  font-weight: 300;
+}
+
+body {
+  padding: 0;
+}
+
+.search {
+  margin-top: 20px;
+}
+
+.search__input {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  box-sizing: border-box;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search__input--big {
+  height: 70px;
+  font-size: 16px;
+  background-size: 19px;
+  padding-left: 62px;
+  line-height: 22px;
+  background-position: 25px center;
+}
+
+.search__input--small {
+  height: 42px;
+  font-size: 14px;
+  background-size: 11px;
+  padding-left: 33px;
+  line-height: 19.12px;
+  background-position: 12px 14px;
+}
+
+.search__input::placeholder {
+  color: #3d4e61;
+}
+
+.search__input:hover {
+  box-shadow: 0 3px 8px 0 #00000040;
+}
+
+.search__input:focus {
+  font-weight: 900px;
```

Comment:
The `font-weight` property value '900px' is incorrect. Font-weight should be a numeric value (e.g., 400, 700) or keywords like 'bold', not a value with 'px'. Consider changing it to a valid font-weight value.